ui/store/actions.js

```diff
@@ -2786,40 +2812,25 @@ export async function updateTokenType(tokenAddress) {
 }
 
 /**
- * initiates polling for gas fee estimates.
+ * Makes a request for, and then starts a timer to continue to make requests
+ * for, a piece of data.
  *
- * @returns {string} a unique identify of the polling request that can be used
- *  to remove that request from consideration of whether polling needs to
- *  continue.
+ * @param {string} item - The type of request to add to the polling queue (one
+ * of "gasFeeEstimates" or "isNetworkCongested").
  */
-export function getGasFeeEstimatesAndStartPolling() {
-  return promisifiedBackground.getGasFeeEstimatesAndStartPolling();
+export function updateWithAndStartPollingFor(item) {
```

Comment:
These changes come from the corresponding commit in the `controllers` repo: https://github.com/MetaMask/controllers/pull/609